settings.py

```diff
@@ -54,10 +54,15 @@
 # will be the path to the `stylus` and `lessc` executables.
 STYLUS_BIN = os.getenv('STYLUS_BIN', path('node_modules/stylus/bin/stylus'))
 LESS_BIN = os.getenv('LESS_BIN', path('node_modules/less/bin/lessc'))
-CLEANCSS_BIN = os.getenv('CLEANCSS_BIN',
-                         path('node_modules/clean-css/bin/cleancss'))
-UGLIFY_BIN = os.getenv('UGLIFY_BIN',
-                       path('node_modules/uglify-js/bin/uglifyjs'))
+CLEANCSS_BIN = os.getenv(
+    'CLEANCSS_BIN',
+    path('node_modules/clean-css/bin/cleancss'))
```

Comment:
This is maybe the best explanation/reason i've seen. You just convinced me.

So now, let's flake8/enforce this thing (one day, maybe).